browser/ui/webui/settings/default_brave_shields_handler.cc

```diff
@@ -46,9 +46,9 @@ void DefaultBraveShieldsHandler::RegisterMessages() {
           &DefaultBraveShieldsHandler::SetFingerprintingControlType,
           base::Unretained(this)));
   web_ui()->RegisterMessageCallback(
-      "setHTTPSEverywhereControlType",
+      "setHTTPSEverywhereEnabled",
```

Comment:
the `controlType` suffix makes it easy to spot along with other sibling methods. can we keep this naming convention?

Reply:
I specifically changed it because ControlType was for allow/block/block_third_party. Changing to Enabled was more consistent with using a boolean and I wanted them to be different so they didn't get confused with each other. I think it would be more confusing to use a boolean for some `set*ControlType` methods and an actual ControlType for others